02week/pigLatin.js

```diff
@@ -9,8 +9,39 @@ const rl = readline.createInterface({
 
 
 const pigLatin = (word)  => {
+  
+    const wordtrim = word.trim().toLowerCase();
+    // console.log(wordtrim);
+    const vowels = ['a' , 'e' , 'i' , 'o', 'u'];
+    // console.log(vowels);
+    // console.log(wordtrim.length)
+    for (let i=0; i < wordtrim.length; i++){
+       for (let j=0; j < vowels.length; j++) {
```

Comment:
you could have gotten away with `vowels.includes(wordtrim.charAt(i))` instead of the inner for loop.